.ci/vfio_jenkins_job_build.sh

```diff
@@ -181,7 +181,7 @@ create_config_iso() {
 pull_fedora_cloud_image() {
 	fedora_img="$1"
 	fedora_img_cache="${fedora_img}.cache"
-	fedora_version=32
+	fedora_version=34
 
 	if [ ! -f "${fedora_img_cache}" ]; then
 		curl -sL "https://download.fedoraproject.org/pub/fedora/linux/releases/${fedora_version}/Cloud/${arch}/images/Fedora-Cloud-Base-${fedora_version}-1.6.${arch}.raw.xz" -o "${fedora_img_cache}.xz"
```

Comment:
image not found, please update also the last part  **Fedora-Cloud-Base-${fedora_version}-1.2.${arch}.raw.xz**